siobrultech_protocols/gem/protocol.py

```diff
@@ -15,10 +16,22 @@
 LOG = logging.getLogger(__name__)
 
 PACKET_HEADER = bytes.fromhex("feff")
+API_RESPONSE_WAIT_TIME_SECONDS = 3.0  # Time to wait for an API response
+PACKET_DELAY_CLEAR_TIME_SECONDS = 3.0  # Time to wait after a packet delay request so that GEM can finish sending any pending packets
 
 
 class PacketProtocol(asyncio.Protocol):
-    def __init__(self, queue: asyncio.Queue):
+    """Protocol implementation for processing a stream of data packets from a GreenEye Monitor."""
+
+    def __init__(
+        self,
+        queue: asyncio.Queue,
+    ):
+        """
+        Create a new protocol instance.
+
+        Whenever a data packet is received fromthe remote GEM, a `Packet` instance will be enqueued to `queue`.
```

Comment:
nit: "from the"

Reply:
Also, it's not necessarily a remote GEM, as in the case of a serial connection.  The `Protocol` doesn't care where it is :D

Reply:
It's on the remote end of a serial cable, innit? ;)

Reply:
still need to add a space between "from" and "the" here